test/units/cli/test_galaxy.py

```diff
@@ -556,6 +563,29 @@ def test_collection_skeleton(collection_skeleton):
         assert expected_contents == contents.strip()
 
 
+@pytest.fixture()
+def collection_build(collection_skeleton, reset_cli_args):
+    output_dir = tempfile.mkdtemp()
```

Comment:
Same: just use a built-in pytest fixture rather than creating dirs outside of the tmp context of pytest session.